frontend/assets/stylesheets/state/_responsive.scss

```diff
@@ -8,18 +9,21 @@
     }
 }
 
-.hidden-mobile-portrait {
+.hidden-mobile-portrait,
+.u-hidden-until-mobileLandscape {
```

Comment:
Trying to improve the semantics of these classnames as it's a) not that explicit what they do b) because of the ambiguous names we already have some overlap, e.g. hidden-mobile and hidden-tablet are the same.